def error_func():
    with open('abc.txt', 'r') as file:
        pass